README.md

```diff
@@ -62,6 +62,21 @@ ng build
 The build artifacts will be stored in the `dist/` directory.
 
 
+### Running tests
+
+Before running the tests make sure that the project is built. To build the 
+project once you can use:
+
+```bash
+ng build
+```
+
+With the project built in the `dist/` folder you can just run: `karma start`. 
+Karma will run the tests and keep the browser open waiting to run again.
+
+To watch the files and build them after any change you can use `ng serve`.
```

Comment:
I don't think that this is true. We'll need to copy DestCopy broccoli plugin from angular/angular into the pipeline to make this work. am I wrong?